content/cloud/data-redis.md

```diff
@@ -59,32 +59,24 @@ The template we are interested in is `http-rust`.
 
 ## Creating Our New Spin Application
 
-The official Spin CLI documentation also has instructions on how to [create a new Spin application](/cloud/cli-reference#new). Let's go ahead and create a new `http-rust` application: 
+The official Spin CLI documentation also has instructions on how to [create a new Spin application](/cloud/cli-reference#new). Let's go ahead and create a new `http-rust` application (use up/down arrow keys to select `http-rust` when prompted):
 
 <!-- @selectiveCpy -->
 
 ```bash
-$ spin new http-rust redisRustApplication
-```
-
-The above command will ask you to enter a `Project description`, `HTTP base` (default set to `/` which is fine) and `HTTP path`:
-
-<!-- @nocpy -->
+$ spin new
 
-```bash
-Project description: 
-HTTP base: /
-HTTP path: /...
+> http-rust (HTTP request handler using Rust)
 ```
 
-You can fill out these values, when prompted, for example:
+The above command will ask you to enter a `name`, `description` and `HTTP path` (hit enter to accept the default `/...` HTTP path value when presented):
 
 <!-- @nocpy -->
 
 ```bash
-Project description: A new redis-rust Spin Application.
-HTTP base: /
-HTTP path: /data
+Enter a name for your new application: redisRustApplication
+Description []: A new redis-rust Spin Application.
```

Comment:
This description risks confusion with the redis-rust (Redis pub-sub trigger) template.